src/command/command/legal-entity/register-legal-entity.command.ts

```diff
@@ -1,3 +1,12 @@
 import { LegalEntityCommand } from './legal-entity.command';
 
-export class RegisterLegalEntityCommand extends LegalEntityCommand {}
+export class RegisterLegalEntityCommand extends LegalEntityCommand {
```

Comment:
Zal ik in een aparte PR deze hernoemen naar `RegisterOrganizationCommand`? Dan mergen we eerst deze. OK?

Reply:
is prima, goed idee